### PR TITLE
Test cases for API Sandbox service 

### PR DESCRIPTION
**Pull request by IrinaComindaru**
https://github.com/capsulajs/api-sandbox/issues/4

### CODE REVIEW

File: tests/features/environment.feature [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+  Scenario: Call selectEnv with invalid environment name
+   Given API Sandbox Service with selectEnv method
+   And   user received several environments from environments$ subscription
+   When  user calls selectEnv method by providing one of the following <envName>
+     	 |envName        |
+         |null           |
+         |undefined      |
+         |123            |
+         |'test'         |
```
Comment (MikeSquall) [inline tests/features/environment.feature]: Actually, `'test'` could be a valid envName

File: tests/features/method.feature [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+  Scenario: Call selectMethod with invalid serviceName
+   Given API Sandbox Service with selectMethod method
+   And   user received several service methods from methods$ subscription
+   When  user calls selectMethod with a valid methodName one of the following <serviceName>
+     	 |serviceName    |
+         |null           |
+         |undefined      |
+         |123            |
+         |'test'         |
```
Comment (MikeSquall) [inline tests/features/method.feature]: serviceName is a `string` so this could be a good value

File: tests/features/method.feature [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+  Scenario: Call selectMethod with invalid methodName
+   Given API Sandbox Service with selectMethod method
+   And   user received several service methods from methods$ subscription
+   When  user calls selectMethod with a valid serviceName one of the following <methodName>
+     	 |methodName     |
+         |null           |
+         |undefined      |
+         |123            |
+         |'test'         |
```
Comment (MikeSquall) [inline tests/features/method.feature]: methodName is a `string` so this could be a good value

File: tests/features/environment.feature (hunk 1 above)
Comment (IrinaComindaru) [reply tests/features/environment.feature]: fixed

File: tests/features/logs.feature [hunk 4]
```diff
@@ -0,0 +1,16 @@
+
```
Comment (MikeSquall) [inline tests/features/logs.feature]: Logs are related to `invoke`, could you move the tests from here to `invoke.feature` ?

**Review by MikeSquall — APPROVED** [review]

**Pull request merged**